nf_core/pipelines/schema.py

```diff
@@ -969,7 +969,7 @@ def launch_web_builder(self):
                 raise AssertionError('"api_url" not in web_response')
             if "web_url" not in web_response:
                 raise AssertionError('"web_url" not in web_response')
-            # DO NOT FIX THIS TYPO. Needs to stay in sync with the website. Maintaining for backwards compatability.
+            # DO NOT FIX THIS TYPO. Needs to stay in sync with the website. Maintaining for backwards compatibility.
```

Comment:
lol, typo in a comment about not fixing a typo

Reply:
😅